### DISCUSSION TITLE
[Feature] date format in dd-mm-yyyy

### DISCUSSION THREAD

**Discussion by jack2game** [johanohly/AirTrail#61] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently I can only use mm-dd-yyyy

**Comment by johanohly** [comment]
Thank you for creating this feature request. I have received similar requests, but after yours I decided to investigate if it would be that hard at all. 
I just found out that this is totally possible. It will almost certainly be available in the next release (automatically decided via your browser preference).

**Comment by johanohly** [comment]
Added in v0.3.1!

**Comment by jack2game** [comment]
Thank you very much. Works like a charm!
